dpc-sdp/ripple-framework
# Feature request: Multi-step forms

**Issue**:
This is a WIP PR to demonstrate a basic integration of multi-step forms, using the Webform element type `webform_wizard_page` in the backend and `@formkit/addons` in the frontend. Basic styling is added by the frontend package, as well as step navigation & validation indicators.

Implements: #1170 

### What I did

- Added `@formkit/addons` package
- Mapped backend `webform_wizard_page` element type to the addon's `step` Formkit type
- Un-nested step form data in the `submitHandler` for `RplForm`

### How to test

- Enable the 'Wizard page' Webform element type in the backend at `/admin/structure/webform/config/elements`
- Create a test webform using the wizard page element & place it on a test landing page
- Update frontend packages `ripple-ui-forms` & `ripple-tide-webform` to use this branch
- Run dev on the frontend and view the test landing page
- Confirm form steps are shown and form data submits correctly

### Checklist




#### For all PR's


#### For new components only
